quantities/quantities/Data.cs

```diff
@@ -26,35 +26,35 @@ namespace Quantities.Quantities;
 {
     private static readonly Polynomial bytes = Polynomial.Of<Units.Si.Metric.Byte>();
     private static readonly IRoot root = new MetricRoot<Units.Si.Metric.Byte>();
-    private readonly Quant quant;
-    internal Quant Quant => this.quant;
-    public Factory<To> To => new(new To(in this.quant));
-    private Data(in Quant quant) => this.quant = quant;
-    Quant IQuantity<Data>.Value => this.quant;
+    private readonly Quantity data;
```

Comment:
again kudos for renaming this field :smile:

Reply:
Thanks :-)